jet/hazelcast-3-connector/hazelcast-3-connector-jobs/src/main/java/com/hazelcast/samples/jet/hz3connector/EnrichUsingMapFromHz3Example.java

```diff
@@ -0,0 +1,94 @@
+            "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+                    + "<hazelcast-client xmlns=\"http://www.hazelcast.com/schema/client-config\"\n"
+                    + "                  xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n"
+                    + "                  xsi:schemaLocation=\"http://www.hazelcast.com/schema/client-config\n"
+                    + "                  http://www.hazelcast.com/schema/client-config/hazelcast-client-config-3.12" +
+                    ".xsd\">\n"
+                    + "\n"
+                    + "    <network>\n"
+                    + "        <cluster-members>\n"
+                    + "            <address>127.0.0.1:3210</address>\n"
+                    + "        </cluster-members>\n"
+                    + "    </network>\n"
+                    + "</hazelcast-client>\n";
+
+    public static void main(String[] args) {
+        System.setProperty(ClusterProperty.PROCESSOR_CUSTOM_LIB_DIR.getName(),
+                new File("target/source").getAbsolutePath());
+
+        HazelcastInstance hz = Hazelcast.bootstrappedInstance();
+
+        StreamSource<Trade> tradesSource = tradeStream(2, 2);
+
+        ServiceFactory<Hz3MapAdapter, AsyncMap<String, String>> hz3MapSF =
+                hz3MapServiceFactory("tickers", HZ3_CLIENT_CONFIG);
+
+        ServiceFactory<String, Map<String, String>> sf = ServiceFactory
+                .withCreateContextFn(context -> "context")
+                .withCreateServiceFn((context, think) ->
+                        new HashMap<>()
+                );
+        Pipeline p = Pipeline.create();
+        BiFunctionEx<? super Map<String, String>, ? super Trade, Tuple3<String, String, Long>> mapFn =
+                Hz3Enrichment.mapUsingIMap(
+                        Trade::getTicker,
+                        (trade, o) -> tuple3(o, trade.getTicker(), trade.getPrice())
+                );
+
+//        BiFunctionEx<? super Map<String, String>, ? super Trade, Tuple3<String, String, Long>> mapFn =
+//                (kvMap, t) -> ((BiFunctionEx<? super Trade, ? super String, ? extends Tuple3<String, String, Long>>) (trade, o) -> tuple3(o, trade.getTicker(), trade.getPrice())).apply(t, kvMap.get(((FunctionEx<? super Trade, ? extends String>) Trade::getTicker).apply(t)));
+
+        StreamStage<Tuple3<String, String, Long>> mapStage =
+                p.readFrom(tradesSource)
+                 .withoutTimestamps()
+                 .mapUsingService(hz3MapSF,
+                         mapFn
+                 );
+        mapStage.writeTo(Sinks.logger());
+
+        JobConfig config = new JobConfig();
+        config.addCustomClasspath(mapStage.name(), "hazelcast-3.12.12.jar");
+        config.addCustomClasspath(mapStage.name(), "hazelcast-client-3.12.12.jar");
+        config.addCustomClasspath(mapStage.name(), "hazelcast-3-connector-impl-5.0-SNAPSHOT.jar");
+        hz.getJet().newJob(p, config);
+
```

Comment:
Could we print a simple message that everything is done like in the previous example?